packages/opentelemetry-core/src/platform/node/log-level.ts

```diff
@@ -0,0 +1,26 @@
+/**
+ * Copyright 2020, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Returns an environmentally set log level if set.
+ * Defaults to INFO.
+ * @returns the log level for opentelemetry.
+ */
+import { LogLevel, LogLevelString } from '../../common/types';
+
+export function getLogLevel(): LogLevel {
+  return LogLevel[(process.env.OTEL_LOG_LEVEL as LogLevelString) ?? 'INFO'];
```

Comment:
if an invalid string is provided, this will return undefined.
```suggestion
  return LogLevel[(process.env.OTEL_LOG_LEVEL as LogLevelString)] ?? LogLevel.INFO;
```

Reply:
Ah yes good catch!